mpu6050/dt/compile.sh

```diff
@@ -0,0 +1 @@
+olegh@orangepione:/boot/overlay-user$ dtc -I dts -O dtb mpu6050.dtso > mpu6050.dtbo
```

Comment:
Does it work as expected?

Reply:
If you mean did overlays work - yes, sure, it works.
If you mean that there is no dtso file in the pull request - yes, it's an missmatch and my little error. After some experiments I had changed file name to .dts, but this example had been left unchanged - i was made before. It was long time ago, more than a month.